### PR TITLE
Adds arg for updater instance type

### PR DESCRIPTION
**Pull request by dramdass**
I ran into an issue when updating an AMI on behalf of a customer
where AWS did not have the the capacity to launch c3.large
instances for more than an hour. As a workaround I had to hack
the brkt cli to use an m3.medium instead. We should add an
instance type arg for anywhere we launch an instance so we
aren't beholden to AWS capacity issues.

Makes the default updater type to m3.medium. The previous was
c3.large to match the encryptor instance type. However, we
don't need a beefy instance type for updating since we don't
do any resource intensive operations during updating.

### CODE REVIEW

File: brkt_cli/aws/update_encrypted_ami_args.py [hunk 1]
```diff
@@ -35,7 +35,16 @@ def setup_update_encrypted_ami(parser):
...
+            'The instance type to use when running the encrypted guest '
+            'instance'),
+        default='m3.medium'
+    )
+    parser.add_argument(
+        '--updater-instance-type',
+        metavar='TYPE',
+        dest='updater_instance_type',
+        help=(
+            'The instance type to use when running the updater '
```
Comment (bburtin) [inline brkt_cli/aws/update_encrypted_ami_args.py]: Does the usage output automatically print the default for you?

Comment (bburtin) [inline brkt_cli/aws/update_encrypted_ami_args.py]: I was lazy.  Looks like it doesn't.  Can you document the default values for `--guest-instance-type` and `--updater-instance-type`?

**Comment by bburtin** [comment]
Looks good.  For extra credit, you may want to squash the commits. :shipit:

**Comment by dramdass** [comment]
going to push to a new branch with clean commit history

**Pull request closed without merging**